Header header
float64 position
int32 rawPosition
